src/pages/console/formbuilder/overview.mdx

```diff
@@ -32,7 +32,7 @@ Your forms will contain inputs for each field of your data model. Auto-generated
 
 Alternatively, Amplify Studio allows you to build forms completely from scratch. To generate a basic, brand-new form:
 
-1. Navigate to **Studio Console > UI Library**
+1. Navigate to **[Amplify Studio Console](/console/adminui/start/#logging-in-and-creating-an-app) > UI Library**
```

Comment:
I would phrase these as:

> In [Amplify Studio Console](/console/adminui/start/#logging-in-and-creating-an-app), navigate to **UI Library.**

This sets the place as Amplify Studio Console, the navigation menu item as UI Library.